assets/scss/components/_form.scss

```diff
@@ -21,3 +25,230 @@
+
+    small {
+        color: $color-red-300;
+        font-weight: 400;
+    }
+}
+
+.js-form-field-error {
+    label,
+    .js-inline-message {
+        color: $color-red-300;
+    }
+}
+
+.js-inline-message {
+    font-family: $font-sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.5;
+}
+
+.js-form-field-radio {
+    .js-input[type='radio'] {
+        display: none;
+    }
+
+    label:not(:first-child) {
+        padding-left: 32px;
+        position: relative;
+        cursor: pointer;
+
+        &::before {
+            content: '';
+            position: absolute;
+            left: 0;
+            top: -2px;
+            width: 24px;
+            height: 24px;
+            border-radius: 50%;
+            background: $color-white;
+            border: 1px solid $color-neutrals-75;
+            transition: all 0.2s ease-in-out;
+        }
+
+        &:hover {
+            &::before {
+                border: 1px solid $color-primary-300;
+                transition: all 0.2s ease-in-out;
+            }
+        }
+
+        &:focus {
+            &::before {
+                box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+                transition: all 0.2s ease-in-out;
+            }
+        }
+    }
+
+    .js-input[type='radio'] {
+        &:checked {
+            + label {
+                &::before {
+                    background-color: $color-white;
+                    border: 6px solid $color-primary-300;
+                }
+
+                &:hover {
+                    &::before {
+                        border-color: $color-primary-400;
+                    }
+                }
+
+                &:focus {
+                    &::before {
+                        box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+                    }
+                }
+            }
+        }
+    }
+}
+
+.js-form-field-checkbox {
+    .js-input[type='checkbox'] {
+        display: none;
+    }
+
+    label {
+        &:not(:first-child) {
+            padding-left: 32px;
+            position: relative;
+            cursor: pointer;
+            display: block;
+        }
+
+        &::before {
+            content: '';
+            position: absolute;
+            left: 0;
+            top: 0;
+            width: 24px;
+            height: 24px;
+            border-radius: 4px;
+            background: $color-white;
+            border: 1px solid $color-neutrals-75;
+            transition: all 0.2s ease-in-out;
+        }
+
+        &:hover {
+            &::before {
+                border-color: $color-primary-400;
+                transition: all 0.2s ease-in-out;
+            }
+        }
+
+        &:focus {
+            &::before {
+                box-shadow: 0px 0px 0px 4px rgba(90, 100, 193, 0.24);
+                transition: all 0.2s ease-in-out;
+            }
+        }
+    }
+
+    .js-input[type='checkbox'] {
+        &:checked {
```

Comment:
this could be moved to line 189 and prevent duplicating selector `.js-input[type='checkbox']`